docs/static/tab-widgets/install-agent.asciidoc

```diff
@@ -1,10 +1,5 @@
 // tag::fleet-managed[]
-. When the **Add Agent flyout** appears, stay on the **Enroll in fleet** tab
-+
---
-[role="screenshot"]
-image::../monitoring/images/integration-agent-add.png[Add agent flyout in {kib}]
---
```

Comment:
@karenzone note to self: Note that you removed references, but did not delete image files from the repo